src/components/Box.js

```diff
@@ -3,7 +3,7 @@ import { compose, space, color, layout, flexbox } from 'styled-system';
 import { createPropTypes } from '@styled-system/prop-types';
 
 // https://styled-system.com/guides/build-a-box
-const Box = styled.div`
+export const Box = styled.div`
   ${compose(space, color, layout, flexbox)}
 
   // ensures the Box can shrink below its minimum content size when used as a flex item
```

Comment:
Looks like this is a superfuious Box component?

Reply:
In that we only use the component once or because I forgot to delete this file when cloning to `Box/Box.js`?

Reply:
the second one 😉

Reply:
:sweat_smile: thanks.